src/value_generators/src/DocumentGenerator.cpp

```diff
@@ -1901,6 +2011,10 @@ const auto [allParsers, arrayParsers, dateParsers, doubleParsers, intParsers, st
          [](const Node& node, GeneratorArgs generatorArgs) {
              return std::make_unique<IncGenerator>(node, generatorArgs);
          }},
+        {"^ConvertToInt",
```

Comment:
Shoudl this be `ConvertToInt64t`?

Reply:
I think if I did this, it would be correct to also rename `^RandomInt` to `^RandomInt64`, but I'm reluctant to do that as it would be a breaking change. We could add an alias `^RandomInt64` and deprecate `^RandomInt` (throwing some type of warning at load time), but that feels hefty. Changing all the things named `intXXX` internally to `int64XXX` (i.e. `intParsers` to `int64Parsers` as you mentioned below) sounds good to me though.